### PR TITLE
add tips on login page

### LINKED ISSUE #1488: Add tips on the Login page
# Feature request

## Please describe your feature

Add tips on the Login page, so users could know how to change the default password & username.

## Describe the solution you'd like

1. Modify the Login page;
2. Added some tips;

### PR DESCRIPTION
**Pull request by Baoyuantop**
Please answer these questions before submitting a pull request

**Why submit this pull request?**
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**Related issues**
resolve #1488

### CODE REVIEW

File: web/src/pages/User/locales/en-US.ts [hunk 1]
```diff
@@ -25,6 +25,8 @@ export default {
   'component.user.loginMethodPassword.incorrectPassword': 'Incorrect username or password',
   'component.user.loginMethodPassword.fieldInvalid': 'Please check username and password',
   'component.user.loginMethodPassword.success': 'Login Success',
+  'component.user.loginMethodPassword.changeDefaultAccount': 'How to update username/password ?',
```
Comment (juzhiyuan) [inline web/src/pages/User/locales/en-US.ts:28]:
```suggestion
  'component.user.loginMethodPassword.changeDefaultAccount': 'How to update username/password?',
```

File: web/src/pages/User/locales/zh-CN.ts [hunk 2]
```diff
@@ -25,6 +25,8 @@ export default {
   'component.user.loginMethodPassword.incorrectPassword': '账号或密码错误',
   'component.user.loginMethodPassword.fieldInvalid': '请检查账号和密码',
   'component.user.loginMethodPassword.success': '登录成功',
+  'component.user.loginMethodPassword.changeDefaultAccount': '如何修改默认账户和密码 ？',
```
Comment (juzhiyuan) [inline web/src/pages/User/locales/zh-CN.ts:28]:
```suggestion
  'component.user.loginMethodPassword.changeDefaultAccount': '如何修改默认账户和密码？',
```

**Review by juzhiyuan — APPROVED** [review]
This is a Tip PR

**Comment by juzhiyuan** [comment]
@Baoyuantop Hi, could you please take some screenshots on Login page?

**Comment by Baoyuantop** [comment]
截屏2021-02-21 下午4 48 08
截屏2021-02-21 下午4 48 35

**Comment by juzhiyuan** [comment]
image

@Baoyuantop It could be more closer :)

**Review by juzhiyuan — APPROVED** [review]

**Comment by Baoyuantop** [comment]
I will continue to improve in future PR

**Review by LiteSun — APPROVED** [review]

**Pull request merged**